_data/glossary/kubernetes-api.yaml

```diff
@@ -0,0 +1,10 @@
+id: kubernetes-api
+name: Kubernetes API
+tags:
+- fundamental
+- architecture
+short-description: |
+  The application which serves the Kubernetes HTTP API and stores the state of the cluster.
```

Comment:
"The application that serves Kubernetes functionality through a RESTful interface and stores the state of the cluster."

I'm open to a different description, but I'd prefer to avoid the tautology of defining an API as an API. 😎

Reply:
makes sense! :P